cypress/integration/nx/general.spec.ts

```diff
@@ -0,0 +1,12 @@
+describe('Default site', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('loads home page', () => {
+    cy.findByRole('heading', { name: 'Welcome to nx-nextjs-monorepo!' })
+
+    cy.visit('//')
```

Comment:
Why `//`?

Reply:
This was to test that trailing slashes work as expected (aka default removes it)